fft/benches/criterion_fft.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)] // clippy has false positive in benchmarks
```

Comment:
It's not really false positives but a config where these lines actually don't get built: https://github.com/lambdaclass/lambdaworks/pull/268/files#diff-bd200755e9353b007a0dc108f89d6f18a344836104054d74793c37669718d24cR74-R104

I think it's OK to fix it like this, just pointing out it's not a bug or anything.

Reply:
I removed it and clippy doesn't show any warnings 🤔